parm/WaterInTheWestC384NoNest.conf

```diff
@@ -0,0 +1,176 @@
+# This is a UNIX conf file that contains ALL information relating to
+# the HAFS stretched globe configuration with no nest.
+#
+[config]
+# Specify the domain center Latitude and Longitude
+domlat=32.5
+domlon=-135.0
+# Same as domlat and domlon but vitals formatted
+vitalsdomlat=325N
+vitalsdomlon=1350W
+
+# HAFS launcher requires vitals and a storm. This is a placeholder for the global static nest.
+fakestormid=00L
+fakestormname=NATL
+fakestormorg=NHC
+fakestorm_vitals={fakestormorg} {fakestormid} {fakestormname} {YMD} {HH}{min} {vitalsdomlat} {vitalsdomlon} -99 -99 -999 -999 -099 -9 -99 -999 -999 -999 -999 M
+
+run_atm_mvnest=no
+run_wave=no
+ww3_model=ww3
+run_ocean=no
+ocean_model=hycom
+
+run_atm_init=no
+run_atm_init_fgat=no
+run_atm_init_ens=no
+run_atm_merge=no
+run_atm_merge_fgat=no
+run_atm_merge_ens=no
+run_atm_vi=no
+run_atm_vi_fgat=no
+run_atm_vi_ens=no
+run_gsi=no
+gsi_d01=no
+gsi_d02=no
+run_analysis_merge=no
+run_analysis_merge_ens=no
+run_fgat=no
+run_envar=no
+run_ensda=no
+run_enkf=no
+
+[dir]
+PARMforecast={PARMhafs}/forecast/globnest
+
+[grid]
+CASE=C384            ;; FV3 resolution
+LEVS=65              ;; Model vertical levels: 65
+gtype=stretch        ;; grid type: uniform, stretch, nest, or regional
+# If gridfixdir is provided and the dir exists, then use the pre-generated static grid fix files under gridfixdir
+#gridfixdir=/let/hafs_grid/generate/grid
+#gridfixdir=/scratch1/BMC/zrtrr/Samuel.Trahan/westwater/HAFS-weird-C768
+#gridfixdir=/scratch1/BMC/zrtrr/Samuel.Trahan/westwater/C768_nest
+#gridfixdir=/scratch1/BMC/zrtrr/Samuel.Trahan/westwater/C768_nest
+# Otherwise, will generate the model grid according to the following grid parameters
+# Need for grid types: stretch, nest and regional
+stretch_fac=1.0001   ;; Stretching factor for the grid
+target_lon={domlon}  ;; center longitude of the highest resolution tile
+target_lat={domlat}  ;; center latitude of the highest resolution tile
+# Need for grid types: nest and regional
+nest_grids=0
+parent_grid_num=1
+parent_tile=6
+refine_ratio=4
+istart_nest=25
+jstart_nest=73
+iend_nest=744
+jend_nest=648
+regional_esg=no
+
+[forecast]
+ccpp_suite_regional=FV3_global_nest_v0
+ccpp_suite_glob=FV3_global_nest_v0
+ccpp_suite_nest=FV3_global_nest_v0
```

Comment:
Is this needed in the no nest config?

Reply:
I haven't tried removing it.

Reply:
If I reduce this file to only overridden values, those lines will be gone.